config/default.json

```diff
@@ -3,7 +3,6 @@
   "port": 3030,
   "mongodb": "mongodb://localhost:27017/bitcentive",
   "public": "../public/",
-  "main": "development.html",
```

Comment:
had to remove this, otherwise the server responds with `development.html` each time an unknown route is visited, this breaks the npm extension.... This change is still not enough since the server is returining 500s for missing package.json files instead of 404s.

Reply:
Just to clarify this for myself ...

@m-mujica removed this because this was causing the server to respond with `development.html` on all 404s.  This was breaking steal loading because when it tries to load `node_modules/can-define/map/index`, it needs to see a 404 to fall back to `node_modules/can-define/map/map`.

We need to make the server respond with `development.html` ONLY for `/`.

Reply:
that was pretty much a copy/paste of what I typed 3 days ago :trollface: .... I'll see myself out